docs/gtfs/Gtfs_Instance_class_diagram.md

```diff
@@ -0,0 +1,25 @@
+# GtfsInstance
+
+<header>
+    <style>
+    div.mermaid {
+      text-align: center;
+    }
+    </style>
+</header>
+
+```mermaid
+
+classDiagram
+    class GtfsInstance {
+        +feed : gtfs_kit.feed
```

Comment:
Colon appears doubled in all lines except `feed` and `get_calendar_dates()` in readme preview.

Reply:
Good spot, not sure what's gone on there. There are some differences in the GitHub mermaid renderring to what I have in VSCode.

Reply:
@SergioRec  This should be fixed now. Turns out that attributes need a colon but methods do not.